### ISSUE TITLE
Ask for consents during instructor training application

### ISSUE THREAD

**Issue by fmichonneau** [carpentries/amy#2162]
When people submit an application for instructor training (https://amy.carpentries.org/forms/request_training/), the consents at the bottom of the page should list all the (non-archived) required consents.

**Comment by pbanaszkiewicz** [comment]
@maneesha @froggleston 

Currently we recognize "consents required for creating a profile":
![obraz](https://user-images.githubusercontent.com/72821/226210487-09037e76-e614-457b-8d00-cf0043df241c.png)

Which of these should be presented to the user on the instructor training request form?

**Comment by maneesha** [comment]
As we discussed today, all consents should be there.  The one for "making profile public" should read:

> Allow posting your name and any public profile you list (website, Twitter) on our website *only after you complete Instructor Training certification*. Emails will not be posted.

**Comment by pbanaszkiewicz** [comment]
@maneesha @froggleston 

I have additional question about existing checkboxes:
![obraz](https://user-images.githubusercontent.com/72821/227766729-0de9c511-4ff7-444b-884d-c545a72fa0cc.png)

I was able to submit a test training request without any of these fields ticked. Is this intended? I can change this behavior to force people to accept these.

**Comment by maneesha** [comment]
For now, please keep the other questions are they are. This is part of the larger changes we want to make to the instructor training application, so we will review this all together in a separate issue/project.

**Comment by froggleston** [comment]
Could this be clarified? Something like:
"The Carpentries will not make public any of your personal data when going
through the application process. However, once certified, do you consent to
your name and any linked social media accounts being published on The
Carpentries website?"

**Comment by pbanaszkiewicz** [comment]
@maneesha @froggleston can you create a new story for that?
